cockpit/src/subscriptions-view.jsx

```diff
@@ -27,11 +27,61 @@ import { InsightsStatus } from './insights.jsx';
 let _ = cockpit.gettext;
 
 class Listing extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            attaching_in_progress: false,
+            attaching_in_progress_promise: null,
+            attaching_message: _("Auto-attach")
```

Comment:
This is actually the button text, not just a temporary progress message, so I would call this `attach_button_text` or similar.
(I don't know about changing the button text to show progress messages... what about a spinner? @andreasn?)